app/src/ui/diff/diff-options.tsx

```diff
@@ -131,13 +130,6 @@ export class DiffOptions extends React.Component<
   }
 
   private renderHideWhitespaceChanges() {
-    if (
-      this.props.sourceTab === RepositorySectionTab.Changes &&
```

Comment:
Do we not need the check for sourceTab === Changes?

Reply:
I can't tell but given the `&& !enableHideWhitespaceInDiffOption()` part, which was always `false`, it was irrelevant. Just checked the comment of the `enableHideWhitespaceInDiffOption` function and it only affected the "Changes" tab, so I think this is correct!

Reply:
Ah ok. That makes sense.